member_database/main.py

```diff
@@ -19,13 +20,21 @@
     send_password_reset_mail,
     load_reset_token
 )
-from .forms import PersonEditForm
+from .forms import PersonEditForm, MembershipForm
 from .mail import send_email
 
 
 main = Blueprint('main', __name__)
 
 
+@main.before_app_first_request
+def init_database():
+    states = ('email_unverified', 'pending', 'confirmed', 'denied', 'canceled')
```

Comment:
How about making this constants, so my IDE tools can remember how things are spelled for me? 😁

Reply:
done